src/qvm.lisp

```diff
@@ -131,6 +131,9 @@ ALLOCATION is an optional argument with the following behavior.
   (bring-to-zero-state (amplitudes qvm))
   qvm)
 
+(defmethod apply-classical-readout-noise ((qvm pure-state-qvm) (instr quil:measurement))
```

Comment:
isn't this method defined much later? When this DEFMETHOD is executed, it's not yet defined. I think it needs to move back to the DEFGENERIC.